integrations/java-iceberg-1.5/openhouse-java-runtime/build.gradle

```diff
@@ -0,0 +1,83 @@
+plugins {
+  id 'openhouse.java-minimal-conventions'
+  id 'openhouse.maven-publish'
+  id 'com.github.johnrengelman.shadow' version '7.1.2'
+}
+
+configurations {
+  fatJarPackagedDependencies {
+    // Following exclusions are not needed during runtime
+    // and often cause conflict with existing classpath.
+    exclude(group: 'org.slf4j') // logging libraries
+    exclude(group: 'org.apache.log4j')
+    exclude(group: 'org.apache.logging.log4j')
+    exclude(group: 'org.mapstruct')
+    exclude(group: 'io.micrometer') // not used in client
+    exclude(group: 'ch.qos.logback')
+  }
+  shadow.extendsFrom implementation
+}
+
+ext {
+  icebergVersion = '1.5.2'
+  sparkVersion = '3.5.2'
+  springVersion = '2.7.8'
+  hadoopVersion = '2.10.0'
+}
+
+dependencies {
+  compileOnly project(':client:secureclient')
+  compileOnly project(':client:tableclient')
+  compileOnly("org.apache.iceberg:iceberg-spark-runtime-3.5_2.12:" + icebergVersion)
+  compileOnly ("org.springframework.boot:spring-boot-starter-webflux:" + springVersion)
+
+  implementation 'org.apache.commons:commons-lang3:3.12.0'
+  fatJarPackagedDependencies(project(':client:secureclient'))
+  implementation("org.apache.iceberg:iceberg-core:" + icebergVersion)
+  implementation("org.apache.hadoop:hadoop-client:" + hadoopVersion) {
+    exclude group: 'junit', module: 'junit'
+    exclude group: 'javax', module: 'servlet-api'
+    exclude group: "io.netty"
+    exclude group: 'com.zaxxer', module: 'HikariCP-java7'
+    exclude group: 'org.apache.commons', module: 'commons-lang3'
+  }
+}
+
+// Following codeblock completely relocates contents of the jar
+// except for source code written in the module. As a result,
+// we remove chances of classpath conflicts during runtime/compiletime.
+shadowJar {
+  dependencies {
```

Comment:
this shadow block will be thin if we reuse code from iceberg 1.2 module

Reply:
How is that? We still need to assemble the uber jar like in the iceberg 1.2 module right?

Reply:
I'm using sourceSet so I still need to copy the shadowJar. The reason I'm not using the `implementation` is that it will cause spark tests to fail.